README.md

```diff
@@ -1 +1,143 @@
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [https://cryptowatch.onrender.com/]
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🛠 Built With <a name="built-with"></a>
+
+- React and redux
+- Visual Studio Code
+- Git & Github
+- CSS
+- Fetch API
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+### Key Features
+
+- Browse a list of available currencies
+- Filter currencies by name
+- View Cryptocurrencies values related to each specific currency
+- Navigate to a details page to view detailed data for each currency
+- Fully responsive design, adapting to different screen sizes and devices
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running follow these simple example steps.
+
+- Open terminal
+- Clone this project by the command `https://github.com/TaiwoEnoch/cryptocurrency-data-explorer.git`
+- `cd <clone>` folder
+- Run `npm start` in your local browser or using Live Server in Visual Studio Code.
+
+### Prerequisites
+
+- IDE to edit and run the code (I used Visual Studio Code 🔥).
+- [Node.js](https://nodejs.org/en/download/) already downloaded.
+- Create React App - [click here to getting started](https://create-react-app.dev/docs/getting-started).
+
+### Setup
+
+Clone this repository to your desired folder:
+`cd my-folder`
+`git clone https://github.com/TaiwoEnoch/cryptocurrency-data-explorer.git`
+
+### Usage
+
+To run the project execute the command below
+
+`npm start`
+
+### Run tests
+
+To run tests, run the following command:
+
+- npx eslint "\*_/_.{js,jsx}"
+
+### Future Features
+
+- Create an interactive web application using React and Redux that allows users to explore Cryptocurrencies data in a variety of currencies.
+- Add a crypto news feed.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 👥 Author
+
+<a name="authors"></a>
+
+👤 **Bilal Ahmed**
+
+- GitHub: [@bilalrajput09](https://github.com/bilalrajput09)
+- Twitter: [@bilal_rajput09](https://twitter.com/bilal_rajput09)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/bilal-ahmed-18b12019a/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project you can give it a ⭐️.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+- Hat tip to [Nelson Sakwa on Behance](https://www.behance.net/sakwadesignstudio) as his design was used 🔰
+- The standup team 🏹⚡
+- My Family 🙌
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
```

Comment:
- [ ] 🙏Please fix the MIT license; it appears that your file is in lowercase (.licence), while the file you're trying to reference is in uppercase (.LICENCE). Kindly ensure that the file📜 naming is consistent. This will allow others👨‍👩‍👧‍👦 to perceive your project as professional and licensed. Also, 🙏please verify that the indentation is correct so that others can easily understand and use your code.

Reply:
Fixed ✔